wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -55,15 +55,6 @@
           "version": "1.0.17"
         }
       },
-      {
-        "package": "libPhoneNumber",
-        "repositoryURL": "https://github.com/wireapp/libPhoneNumber-iOS.git",
-        "state": {
-          "branch": null,
-          "revision": "a6f5cb0a2a264e8396da65b0fb27ecf0df8a975e",
-          "version": "1.1.0"
-        }
-      },
       {
```

Comment:
wanted?

Reply:
I think so, yes.

Reply:
What do you mean? @netbe

Reply:
you should not commit this, please revert.

Reply:
Done: https://github.com/wireapp/wire-ios/pull/1949/commits/bcf34d4ae774176ade488095a72380c0542cc761